ivadomed/inference.py

```diff
@@ -657,6 +658,6 @@ def image_reconstruction(batch: dict, pred: tensor, undo_transforms: UndoCompose
     weight_matrix[:, x_min:x_max, y_min:y_max] += 1
     if last_patch_bool:
         image /= weight_matrix
+        pred_undo, metadata = undo_transforms(image, batch['gt_metadata'][smp_idx], data_type='gt')
```

Comment:
So simply yet so brilliant!
I scratched my head for a while, but cant see why it would not work